cypress.json

```diff
@@ -1,4 +1,5 @@
 {
   "projectId": "dnbm5t",
-  "baseUrl": "https://int.crossroads.net"
+  "baseUrl": "http://127.0.0.1:4000/",
+  "numTestsKeptInMemory": 0
```

Comment:
Does the above line "fix" memory leak with live?

Reply:
Nope. The only workaround I've found is to use cypress run instead of cypress open